apps/nextjs-website/src/lib/types/webinar.ts

```diff
@@ -0,0 +1,10 @@
+import { Speaker } from '@/lib/types/speaker';
+
+export type Webinar = {
+  readonly description: string;
+  readonly speakers?: readonly Speaker[];
```

Comment:
Can a webinar have no speakers?

Reply:
Yes, I imagined the scenario where the speakers are not yet confirmed (and the date is not sure*), but the devPortal's staff needs to start a campaign about the upcoming webinar

Reply:
yep, makes sense, and an empty array can let you handle webinars without speakers.
if the `speakers` property can also be `undefined` we should both handle `undefined` values and empty arrays